pom.xml

```diff
@@ -250,6 +250,20 @@
       <groupId>pentaho-kettle</groupId>
       <artifactId>kettle-engine</artifactId>
       <version>${pdi.version}</version>
+      <exclusions>
```

Comment:
Is the jackson-databind going to be in lib folder ?

The jira descriptions scope of work has conflicting text

```
pme-ee/metadata-editor/libext/pentaho/jackson-databind-2.14.2.jar Pentaho Metadata Editor is bringing in the jackson-databind dependency through kettle-core. It needs to be excluded as there is no usecase for this library in PME
This needs to be updated as mentioned in the next to next point

pme-ee/metadata-editor/libext/pentaho/jackson-databind-2.14.2.jar this is the dependency for pme we need to upgrade to 2.16.2 over here
```
![ppp_5166_pme_jackson-databind_description](https://github.com/user-attachments/assets/36e856af-74ed-477b-93fc-f824bfe75d08)

Reply:
I had clarified this with Ramaiz earlier and had made changes to the description. But later we found out that it was not needed and could be excluded. This discussion was done on a call. 

Jackson-databind will not be in the lib folder, aslo jackson-annotations and jackson-core will not be there

Reply:
yeah just update the scope. I would have a final comment with the final decisions on all.